packages/xo-server-sdn-controller/src/ovsdb-client.js

```diff
@@ -44,6 +46,16 @@ export class OvsdbClient {
     remoteAddress,
     encapsulation
   ) {
+    if (
+      this._adding.find(
+        elem => elem.id === networkUuid && elem.addr === remoteAddress
+      ) !== undefined
+    ) {
+      return
```

Comment:
So what happens here? Shouldn't you throw an error or something? Can you continue whatever process you're in the middle of without finishing this?

Reply:
It's not a problem, it means this element is already being added and doesn't to be added a second time.
The process can be continued.

Reply:
It's already *being* added but do you need to wait for it to be added?

Reply:
No, it's okay to return.